runtime/jcl/common/jclvm.c

```diff
@@ -325,3 +548,71 @@ Java_com_ibm_oti_vm_VM_dumpString(JNIEnv * env, jclass clazz, jstring str)
 		}
 	}
 }
+
+/**
+ * quicksort the list of pointers to statistics entries based on aggregateSize.
+ * @param list
+ * Vector of pointers to statistics entries
+ * @start index of the first element to be sorted
+ * @end index of the last element to be sorted
+ */
+
+static void
+sortStatsTable(J9HeapStatisticsTableEntry **list, IDATA start, IDATA end)
```

Comment:
Could this not be implemented using the C standard qsort?

Reply:
I will give it a try.